src/content/ja/2019/accessibility.md

```diff
@@ -0,0 +1,418 @@
+これらの要素はほとんど使われておらず、使用率は約20％ですが、
+これらはすべてのWebページで必要となるわけではないため、とくに驚くべき統計ではありません。
+
+これらの要素はすべてアクセシビリティサポートを主目的として設計されており、見た目の変化はありません。つまりこれは、既存の要素を安全に置き換えることが可能なので意図しない影響で苦しむことはないでしょう。
+
+<figure>
+  <a href="/static/images/2019/09_Accessibility/fig5.png">
+    <img src="/static/images/2019/09_Accessibility/fig5.png" alt="図5. 色々なHTMLセマンティック要素の利用率。" aria-labelledby="fig5-caption" aria-describedby="fig5-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSG3DTnx7j-YT1hnQpQYjDRD-rCSF1dXbgva-iJQZKdCKIt34ojGMDRhx74fF93CpPg7oGW_C68fWGT/pubchart?oid=708035719&amp;format=interactive">
+  </a>
+  <div id="fig5-description" class="visually-hidden">各要素のバーと20毎に0〜60の範囲のページの割合を表す水平線を含む縦棒グラフ。nav：53.94％、header：54.82％、footer：55.92％、main：18.47％、aside：16.99％、article：22.59％、hr：19.1％、section：36.55%。</div>
+  <figcaption id="fig5-caption" >図5. 色々なHTMLセマンティック要素の利用率。</figcaption>
+</figure>
+
+### ナビゲーションで使われているその他のHTML要素
+
+よく使われているスクリーンリーダーは、ユーザーがリンク、一覧、一覧のアイテム、iframe、それと編集フィールド、ボタン、リストボックスなどのフォームフィールドに素早く飛び、誘導できます。
+図6はこういった要素を使うページの表示頻度を表しています。
+
+<figure>
+  <a href="/static/images/2019/09_Accessibility/fig6.png">
+    <img src="/static/images/2019/09_Accessibility/fig6.png" alt="図6. ナビゲーションで使われるその他のHTML要素。" aria-labelledby="fig6-caption" aria-describedby="fig6-description" width="600" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-src="https://docs.google.com/spreadsheets/d/e/2PACX-1vSG3DTnx7j-YT1hnQpQYjDRD-rCSF1dXbgva-iJQZKdCKIt34ojGMDRhx74fF93CpPg7oGW_C68fWGT/pubchart?oid=389034849&amp;format=interactive">
+  </a>
+  <div id="fig6-description" class="visually-hidden">各要素を示すバーと、25毎に0~100の範囲でページの割合を示す縦棒グラフ。a：98.22％、ul：88.62％、input：76.63％、iframe：60.39％、button：56.74％、select：19.68％、textarea：12.03%。</div>
+  <figcaption id="fig6-caption">図6. ナビゲーションで使われるその他のHTML要素。</figcaption>
+</figure>
+
+### スキップリンク
+
+[スキップリンク](https://webaim.org/techniques/skipnav/)はスクリーンリーダーやキーボードだけを使うユーザーが、メインコンテンツに直接飛ぶことができるようにする、ページ上部に配置されるリンクです。
+これは、ページの上部にあるすべてのナビゲーションリンクとメニューを効率的に「スキップ」します。
+スキップリンクは、スクリーンリーダーを利用していないキーボードユーザーにとってとくに便利です。それは、このようなユーザーは通常他のクィックナビゲーションモード（ランドマークや見出しなど）にアクセスできないためです。
+サンプリングされたページの14.19％にスキップリンクが使われていました。
+
+スキップリンクの動作を試す事ができます！　シンプルにGoogle検索を実行し、検索結果ページが表示されたらすぐに「Tab」キーを押します。図7のような、事前に隠されたリンクが表示されます。
+
+<figure>
+  <a href="/static/images/2019/09_Accessibility/example-of-a-skip-link-on-google.com.png">
+    <img alt="図7. google.comのスキップリンク外観。" aria-labelledby="fig7-caption" aria-describedby="fig7-description" src="/static/images/2019/09_Accessibility/example-of-a-skip-link-on-google.com.png" width="600">
+  </a>
+  <div id="fig7-description" class="visually-hidden">「http archive」を検索するためのGoogle検索結果ページのスクリーンショット。表示される「Skip to main conent」のリンクは青色のフォーカスハイライトと、スキップリンクを示す赤い矢印の付いた黄色のオーバーレイボックスに囲まれ「A skip link on google.com（google.comのスキップリンク）」と表示されます。</div>
```

Comment:
Shoudl `http archive` be `HTTP Archive` as per https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Wiki#meta ?

Also should we explain the English parts to Japanese readers even if leaving them in the original English?

Reply:
Certainly, I reconsidered that this should be completely Japanese because it looks like a use case for people who cannot see images.

to fixed!